### PR TITLE
Modify S5905: LaYC format

### PR DESCRIPTION
**Pull request by guillaume-dequenne-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

File: rules/S5905/python/rule.adoc [hunk 1]
```diff
@@ -1,27 +1,31 @@
+This rule raises an issue when using an ``++assert++`` statement on a tuple literal.
+
 == Why is this an issue?
 
 Using the ``++assert++`` statement on a tuple literal will always fail if the tuple is empty, and always succeed otherwise.
 
 
-The ``++assert++`` statement does not have parentheses around its parameters. Calling ``++assert(x, y)++`` will test if the tuple ``++(x, y)++`` is True, which is always the case.
+The ``++assert++`` statement does not take parentheses around its parameters. Calling ``++assert(x, y)++`` will test if the tuple ``++(x, y)++`` is True, which is always the case.
```
Comment (joke1196) [inline rules/S5905/python/rule.adoc:8]: Small suggestion: we could maybe say why a nonempty tuple is truthy all the time.

**Review by joke1196 — CHANGES_REQUESTED** [review]
Looks good to me. Just added a small suggestion.

**Review by joke1196 — APPROVED** [review]
LGTM! I like the added explanation

File: rules/S5905/python/rule.adoc [hunk 2]
```diff
@@ -1,27 +1,31 @@
+This rule raises an issue when using an ``++assert++`` statement on a tuple literal.
+
 == Why is this an issue?
 
-Using the ``++assert++`` statement on a tuple literal will always fail if the tuple is empty, and always succeed otherwise.
+When tested for truthiness, a sequence or collection will evaluate to `False` if it's empty (its ``++__len__++`` method returns 0) and to `True` if it contains at least one element.
```
Comment (jean-jimbo-sonarsource) [inline rules/S5905/python/rule.adoc:5]: We should use "if it is empty", rather than the contracted "if it's empty"

**Review by jean-jimbo-sonarsource — APPROVED** [review]
NLGTM

**Pull request merged**